### PR TITLE
feat: add org policies to confidential computing example

### PR DESCRIPTION
**Pull request by arthurlapertosa**
This PR adds org policies to the existing confidential computing example.

### CODE REVIEW

**Comment by arthurlapertosa** [comment]
@apeabody Hey, Andrew! Could you please trigger the build on this PR?

**Comment by arthurlapertosa** [comment]
Looks like T`estInstanceSimpleModule` is failing, but all the other tests (including the test related to the changes made by this PR) are passing. @apeabody would you have a clue on why it's failing?

**Comment by apeabody** [comment]
> Looks like `TestInstanceSimpleModule` is failing, but all the other tests (including the test related to the changes made by this PR) are passing. @apeabody would you have a clue on why it's failing?

Hmm, not sure the sudden change in test outcome (perhaps due to new provider version), but will need to be investigated future.

```
TestInstanceSimpleModule 2024-09-12T16:03:59Z command.go:185: Planning failed. Terraform encountered an error while generating this plan.
TestInstanceSimpleModule 2024-09-12T16:03:59Z command.go:185: 
TestInstanceSimpleModule 2024-09-12T16:03:59Z command.go:185: 
TestInstanceSimpleModule 2024-09-12T16:03:59Z command.go:185: Error: project in subnetwork's self_link "https://www.googleapis.com/compute/v1/projects/ci-vm-module-1e38/regions/us-central1/subnetworks/cft-vm-test-hcg8" must match subnetwork_project ""
TestInstanceSimpleModule 2024-09-12T16:03:59Z command.go:185: 
TestInstanceSimpleModule 2024-09-12T16:03:59Z command.go:185:   with module.instance_simple.module.compute_instance.google_compute_instance_from_template.compute_instance[3],
TestInstanceSimpleModule 2024-09-12T16:03:59Z command.go:185:   on ../../../modules/compute_instance/main.tf line 45, in resource "google_compute_instance_from_template" "compute_instance":
TestInstanceSimpleModule 2024-09-12T16:03:59Z command.go:185:   45: resource "google_compute_instance_from_template" "compute_instance" {
TestInstanceSimpleModule 2024-09-12T16:03:59Z command.go:185: 
TestInstanceSimpleModule 2024-09-12T16:03:59Z command.go:185: 
TestInstanceSimpleModule 2024-09-12T16:03:59Z command.go:185: Error: project in subnetwork's self_link "https://www.googleapis.com/compute/v1/projects/ci-vm-module-1e38/regions/us-central1/subnetworks/cft-vm-test-hcg8" must match subnetwork_project ""
TestInstanceSimpleModule 2024-09-12T16:03:59Z command.go:185: 
TestInstanceSimpleModule 2024-09-12T16:03:59Z command.go:185:   with module.instance_simple.module.compute_instance.google_compute_instance_from_template.compute_instance[1],
TestInstanceSimpleModule 2024-09-12T16:03:59Z command.go:185:   on ../../../modules/compute_instance/main.tf line 45, in resource "google_compute_instance_from_template" "compute_instance":
TestInstanceSimpleModule 2024-09-12T16:03:59Z command.go:185:   45: resource "google_compute_instance_from_template" "compute_instance" {
TestInstanceSimpleModule 2024-09-12T16:03:59Z command.go:185: 
TestInstanceSimpleModule 2024-09-12T16:03:59Z command.go:185: 
TestInstanceSimpleModule 2024-09-12T16:03:59Z command.go:185: Error: project in subnetwork's self_link "https://www.googleapis.com/compute/v1/projects/ci-vm-module-1e38/regions/us-central1/subnetworks/cft-vm-test-hcg8" must match subnetwork_project ""
TestInstanceSimpleModule 2024-09-12T16:03:59Z command.go:185: 
TestInstanceSimpleModule 2024-09-12T16:03:59Z command.go:185:   with module.instance_simple.module.compute_instance.google_compute_instance_from_template.compute_instance[0],
TestInstanceSimpleModule 2024-09-12T16:03:59Z command.go:185:   on ../../../modules/compute_instance/main.tf line 45, in resource "google_compute_instance_from_template" "compute_instance":
TestInstanceSimpleModule 2024-09-12T16:03:59Z command.go:185:   45: resource "google_compute_instance_from_template" "compute_instance" {
TestInstanceSimpleModule 2024-09-12T16:03:59Z command.go:185: 
TestInstanceSimpleModule 2024-09-12T16:03:59Z command.go:185: 
TestInstanceSimpleModule 2024-09-12T16:03:59Z command.go:185: Error: project in subnetwork's self_link "https://www.googleapis.com/compute/v1/projects/ci-vm-module-1e38/regions/us-central1/subnetworks/cft-vm-test-hcg8" must match subnetwork_project ""
TestInstanceSimpleModule 2024-09-12T16:03:59Z command.go:185: 
TestInstanceSimpleModule 2024-09-12T16:03:59Z command.go:185:   with module.instance_simple.module.compute_instance.google_compute_instance_from_template.compute_instance[2],
TestInstanceSimpleModule 2024-09-12T16:03:59Z command.go:185:   on ../../../modules/compute_instance/main.tf line 45, in resource "google_compute_instance_from_template" "compute_instance":
TestInstanceSimpleModule 2024-09-12T16:03:59Z command.go:185:   45: resource "google_compute_instance_from_template" "compute_instance" {
TestInstanceSimpleModule 2024-09-12T16:03:59Z command.go:185: 
    terraform.go:536: 
        	Error Trace:	/builder/home/go/pkg/mod/github.com/!google!cloud!platform/cloud-foundation-toolkit/infra/blueprint-test@v0.16.1/pkg/tft/terraform.go:536
        	            				/workspace/test/integration/instance_simple/instance_simple_test.go:38
        	            				/builder/home/go/pkg/mod/github.com/!google!cloud!platform/cloud-foundation-toolkit/infra/blueprint-test@v0.16.1/pkg/tft/terraform.go:638
        	            				/builder/home/go/pkg/mod/github.com/!google!cloud!platform/cloud-foundation-toolkit/infra/blueprint-test@v0.16.1/pkg/tft/terraform.go:670
        	            				/builder/home/go/pkg/mod/github.com/!google!cloud!platform/cloud-foundation-toolkit/infra/blueprint-test@v0.16.1/pkg/utils/stages.go:31
        	            				/builder/home/go/pkg/mod/github.com/!google!cloud!platform/cloud-foundation-toolkit/infra/blueprint-test@v0.16.1/pkg/tft/terraform.go:670
        	Error:      	Should not be: 1
        	Test:       	TestInstanceSimpleModule
```

File: examples/confidential_computing/org_policies.tf [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  source  = "terraform-google-modules/org-policy/google"
+  version = "~> 5.3"
+
+  project_id       = var.project_id
+  policy_for       = "project"
+  constraint       = "constraints/gcp.restrictNonCmekServices"
+  policy_type      = "list"
+  deny             = ["compute.googleapis.com"]
+  deny_list_length = 1
+}
```
Comment (erlanderlo) [inline examples/confidential_computing/org_policies.tf:39]: also add the `restrictCmekCryptoKeyProjects` policy.  although the example has the one project, this helps show how to implement: https://cloud.google.com/kms/docs/cmek-org-policy#project-constraint

**Review by erlanderlo — COMMENTED** [review]
thanks! one additional cmek policy

**Comment by arthurlapertosa** [comment]
@apeabody can you please rerun the build?

File: examples/confidential_computing/org_policies.tf (hunk 1 above)
Comment (arthurlapertosa) [reply examples/confidential_computing/org_policies.tf:39]: Done, thanks!

File: examples/confidential_computing/org_policies.tf [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+module "restrict-cmek-cryptokey-projects-policy" {
+  source  = "terraform-google-modules/org-policy/google"
+  version = "~> 5.3"
+
+  project_id         = var.project_id
+  policy_for         = "project"
+  constraint         = "constraints/gcp.restrictCmekCryptoKeyProjects"
+  policy_type        = "list"
+  allow              = ["projects/${var.project_id}"]
+  allow_list_length = 1
```
Comment (erlanderlo) [inline examples/confidential_computing/org_policies.tf:50]: plz run the cft tooling. I think it'll call `terraform fmt`

Comment (arthurlapertosa) [reply examples/confidential_computing/org_policies.tf:50]: Oh, I forgot to run it again, sorry!

**Review by erlanderlo — APPROVED** [review]
thanks so much. org policies look great!

**Comment by arthurlapertosa** [comment]
@apeabody could you run the build again, please?

**Comment by arthurlapertosa** [comment]
@apeabody same error occurring, on `TestInstanceSimpleModule`. Do you think we can merge this PR anyway?

**Comment by apeabody** [comment]
> @apeabody same error occurring, on `TestInstanceSimpleModule`. Do you think we can merge this PR anyway?

Hi @arthurlapertosa - This upcoming PR should fix the CI Test: https://github.com/terraform-google-modules/terraform-google-vm/pull/430

**Review by apeabody — APPROVED** [review]
Thanks @arthurlapertosa @erlanderlo!

**Pull request merged**